### PR TITLE
Add prune feature to address #97

### PR DESCRIPTION
**Pull request by jpapejr**
This PR adds a Remove button to the Inactive Clusters widget in the dashboard. Clicking on this button for the cluster will remove all records of it from the Clusters and Resources Mongo collections thus removing it from the dashboard (purging). This PR can address #97

### CODE REVIEW

File: imports/ui/components/inactiveClusters/index.js [hunk 1]
```diff
@@ -34,3 +35,22 @@ Template.inactiveClusters.onCreated(function() {
         this.subscribe('clusters.zombie', Session.get('currentOrgId'));
     });
 });
+
```
Comment (dalehille) [inline imports/ui/components/inactiveClusters/index.js]: after the prunes are complete can you also make a call to `updateResourceStats` meteor method?  That should make the cluster and resources numbers in the nav bar update without having to reload the browser

File: imports/ui/components/inactiveClusters/component.html [hunk 2]
```diff
@@ -25,13 +25,15 @@
...
               <th>Last active</th>
+              <th>Action</th>
             </tr>
           </thead>
           <tbody>
             {{#each cluster in zombieClusters}}
             <tr>
               <td><a href="{{pathFor 'cluster.tab' id=cluster.cluster_id tabId='resources'}}">{{getClusterName cluster}}</a></td>
               <td>{{moment cluster.updated}}</td>
+              <td><button class="delete" cluster_id="{{cluster.cluster_id}}">Remove</button></td>
```
Comment (dalehille) [inline imports/ui/components/inactiveClusters/component.html]: can you add the `btn` and `btn-primary` classes here?  That will pull in the bootstrap button classes that we use elsewhere in razeedash

File: imports/ui/components/inactiveClusters/index.js (hunk 1 above)
Comment (jpapejr) [reply imports/ui/components/inactiveClusters/index.js]: 2b8af2d48aedea09ea4ee20dfce37fe114119f79 added to do the requested action.

File: imports/ui/components/inactiveClusters/component.html (hunk 2 above)
Comment (jpapejr) [reply imports/ui/components/inactiveClusters/component.html]: Added in #7b65142

**Pull request merged**